main.go

```diff
@@ -108,6 +108,10 @@ func main() {
 	klog.InitFlags(nil)
 	flag.Parse()
 
+	zaplog := zap.New(zap.UseFlagOptions(&opts))
+	ctrl.SetLogger(zaplog)
+	klog.SetLogger(zaplog)
```

Comment:
Is klog flag options like ```--v=4``` will still work?

Reply:
it should. This is what we need to check, and which requires further investigation. Let's take our time with this PR. I split the fixes/safe changes in https://github.com/openshift-kni/numaresources-operator/pull/378